versioned_docs/version-22.04/migrate/migrate-from-el-to-el.md

```diff
@@ -0,0 +1,201 @@
+---
+id: migrate-from-el-to-el
+title: Migrate from an EL-type OS to another EL-type OS (from a Centreon 18.10 or newer)
+---
+
+## Prerequisites
+
+This procedure only applies if the following conditions are met:
+
+- You wish to migrate from a 64-bit EL-type OS to another supported 64-bit EL-type OS. For instance, if you want to migrate from a CentOS 7 to an Alma 8.
+- Your version of Centreon is 18.10 or newer.
+
+> If your Centreon platform includes a Centreon redundancy system, please
+> contact [Centreon support](https://centreon.force.com).
+
+## Migrating a central server
+
+### Step 1: Install the new server
+
+1. Install your new OS according to the [prerequisites](../installation/prerequisites.md).
+
+2. Install a new Centreon central server from [packages](../installation/installation-of-a-central-server/using-packages.md), until you
+complete the installation process by connecting to the Centreon web interface.
+
+   > Use the same password for the **centreon** user as for the old platform during the web
+   > installation process.
+
+3. Perform software and system updates:
+
+   ```shell
+   dnf update
+   ```
+
+### Step 2 : Synchronize the data
+
+1. Connect to your old Centreon server.
+
+2. Generate a pair of ssh keys for **root**:
+
+   ```shell
+   ssh-keygen -t rsa
```

Comment:
Key pair generated here will go, by default, to `/root/.ssh/id_rsa.pub and /root/.ssh/id_rsa`, maybe we should add the two possible case

- user chooses to use default values => follow procedure as described with id_rsa and id_rsa.pub
- user chooses a custom key pair name => add the rsync command with the proper -e + -i options
`rsync -avz -e "ssh -i /root/.ssh/id_rsa_custom" /etc/centreon root@10.25.26.27:/tmp/`